File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/widgetassertions/CheckBoxes_View.cpp

#include "CheckBoxes_View.h"
#include "CheckBoxes_View.h"
#include <optional>
#include <string>

namespace widgetassertions
{
  bool CheckBoxes_View::getIsMyFlagWithLabelCheckBoxChecked( ) 
  {
    return this->isMyFlagWithLabelCheckBoxChecked;
  }
  bool CheckBoxes_View::getIsMyFlagWithLabelCheckBoxVisible( ) 
  {
    return this->isMyFlagWithLabelCheckBoxVisible;
  }
  bool CheckBoxes_View::getIsMyFlagWithLabelCheckBoxEnabled( ) 
  {
    return this->isMyFlagWithLabelCheckBoxEnabled;
  }
  std::string CheckBoxes_View::getMyFlagWithLabelCheckBoxText( ) 
  {
    return this->MyFlagWithLabelCheckBoxText;
  }
  bool CheckBoxes_View::getIsMyFlagNoLabelCheckBoxChecked( ) 
  {
    return this->isMyFlagNoLabelCheckBoxChecked;
  }
  std::optional<bool> CheckBoxes_View::getIsMyFlagTriStateCheckBoxChecked( ) 
  {
    return this->isMyFlagTriStateCheckBoxChecked == true;
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/widgetassertions/CheckBoxes_View.cs
/// <filename>
///     CheckBoxes_View.cs
/// </filename>


namespace widgetassertions
{
    public class CheckBoxes_View
  {
        private bool isMyFlagWithLabelCheckBoxChecked;
     
        public     virtual bool getIsMyFlagWithLabelCheckBoxChecked()
    {
      return this.isMyFlagWithLabelCheckBoxChecked;
    }
     
        private bool isMyFlagWithLabelCheckBoxVisible;
     
        public     virtual bool getIsMyFlagWithLabelCheckBoxVisible()
    {
      return this.isMyFlagWithLabelCheckBoxVisible;
    }
     
        private bool isMyFlagWithLabelCheckBoxEnabled;
     
        public     virtual bool getIsMyFlagWithLabelCheckBoxEnabled()
    {
      return this.isMyFlagWithLabelCheckBoxEnabled;
    }
     
        private string MyFlagWithLabelCheckBoxText;
     
        public     virtual string getMyFlagWithLabelCheckBoxText()
    {
      return this.MyFlagWithLabelCheckBoxText;
    }
     
        private bool isMyFlagNoLabelCheckBoxChecked;
     
        public     virtual bool getIsMyFlagNoLabelCheckBoxChecked()
    {
      return this.isMyFlagNoLabelCheckBoxChecked;
    }
     
        private bool? isMyFlagTriStateCheckBoxChecked;
     
        public     virtual bool? getIsMyFlagTriStateCheckBoxChecked()
    {
      return this.isMyFlagTriStateCheckBoxChecked == true;
    }
  }
}

